boa_tester/src/edition.rs

```diff
@@ -276,27 +276,45 @@ static FEATURE_EDITION: phf::Map<&'static str, SpecEdition> = phf::phf_map! {
 )]
 #[serde(untagged)]
 pub(crate) enum SpecEdition {
-    /// [ECMAScript 5.1 Edition](https://262.ecma-international.org/5.1)
+    /// ECMAScript 5.1 Edition
+    ///
+    /// <https://262.ecma-international.org/5.1>
     ES5 = 5,
-    /// [ECMAScript 6th Edition](https://262.ecma-international.org/6.0)
+    /// ECMAScript 6th Edition
+    ///
+    /// <https://262.ecma-international.org/6.0>
     ES6,
-    /// [ECMAScript 7th Edition](https://262.ecma-international.org/7.0)
+    /// ECMAScript 7th Edition
+    ///
+    /// <https://262.ecma-international.org/7.0>
     ES7,
-    /// [ECMAScript 8th Edition](https://262.ecma-international.org/8.0)
+    /// ECMAScript 8th Edition
+    ///
+    /// <https://262.ecma-international.org/8.0>
     ES8,
-    /// [ECMAScript 9th Edition](https://262.ecma-international.org/9.0)
+    /// ECMAScript 9th Edition
+    ///
+    /// <https://262.ecma-international.org/9.0>
     ES9,
-    /// [ECMAScript 10th Edition](https://262.ecma-international.org/10.0)
+    /// ECMAScript 10th Edition
+    ///
+    /// <https://262.ecma-international.org/10.0>
     ES10,
-    /// [ECMAScript 11th Edition](https://262.ecma-international.org/11.0)
+    /// ECMAScript 11th Edition
+    ///
+    /// <https://262.ecma-international.org/11.0>
     ES11,
-    /// [ECMAScript 12th Edition](https://262.ecma-international.org/12.0)
+    /// ECMAScript 12th Edition
+    ///
+    /// <https://262.ecma-international.org/12.0>
     ES12,
-    /// [ECMAScript 13th Edition](https://262.ecma-international.org/13.0)
+    /// ECMAScript 13th Edition
+    ///
+    /// <https://262.ecma-international.org/13.0>
     ES13,
     /// The edition being worked on right now.
     ///
-    /// A draft is currently available in <https://tc39.es/ecma262>.
+    /// A draft is currently available at <https://tc39.es/ecma262>.
```

Comment:
Would it be better for this to be worded more like: `A draft is currently available [here](https://tc39.es/ecma262)`?